test/js/index.js

```diff
@@ -3,7 +3,7 @@ import * as fs from "fs";
 import * as path from "path";
 import { loadConfig, svelte } from "../helpers.js";
 
-describe("js", () => {
+describe.only("js", () => {
```

Comment:
Is this intentional?

Reply:
nah, i just keep accidentally committing these as i'm working on stuff